### ISSUE TITLE
Rethinking Pre-training and Self-training

### ISSUE THREAD

**Issue by ghuijo** [One-Day-One-Paper/One-Day-One-Paper#175]
*"Data Augmentation/Pre-training/Self-training"*

### Methodology
* data augmentation
  * 4가지 augmetation 방법 --> for varying the strength of data augmentation
  * Augment-S1: standard flip and crop augmentation; horizontal flips and scale jittering 포함
  * Augment-S2: 여기서부터는 Augment-S1보다 강한 augmentation 적용. AutoAugment
  * Augment-S3: AutoAugment with higher scale jittering
  * Augment-S4: RandAugment with higher scale jittering
* Pre-training
  * study ImageNet pre-trained checkpoints of varying quality
  * use the EfficientNet-B7 architecture as a strong baseline for pre-training
  * 아래 3가지로 진행
  * Rand Init: Model initialized w/ random weights
  * ImageNet Init: Model initialized w/ ImageNet pre-trained checkpoint (84.5% top-1)
  * ImageNet++ Init: Model initialized w/ higher performing ImageNet pre-trained checkpoint (86.9% top-1)
* Self-training
  * a teacher model is trained on the labeled data
  * teacher model generates pseudo labels on unlabeled data
  * a student is trained to optimize the loss on human labels and pseudo labels jointly
 
### Experiments
* the effects of data augmentation
  <image src="https://user-images.githubusercontent.com/67627471/168476427-9180d9e9-ce7e-4d50-9bba-2b713e771fd4.png" width="40%" height="40%" />
* the effects of dataset size on pre-training
  <image src="https://user-images.githubusercontent.com/67627471/168476455-22ebcd4d-5b6d-4cd5-8e65-7b745907573f.png" width="40%" height="40%"/>
* Pre-training hurts performance when stronger data augmentation is used
* More labeled data diminishes the value of pre-training
* Self-training helps in high data/strong augmentation regimes, even when pre-training hurts
  <image src="https://user-images.githubusercontent.com/67627471/168476532-15ac9372-8603-4ff8-8c1d-7b889bffc155.png" width="80%" height="80%"/>
